app/app.py

```diff
@@ -2,12 +2,16 @@
 Dockless origin/destination trip data API
 # try me
 http://localhost:8000/v1/trips?xy=-97.75094341278084,30.276185988411257&flow=destination&mode=all
+
+#TODO:
+- query params for dow, hour day, date
```

Comment:
dow? day of week? is there a gh-issue for this?

Reply:
https://github.com/cityofaustin/transportation-dockless-api/issues/33